examples/bigcommerce/pages/api/categories.ts

```diff
@@ -2,11 +2,6 @@ import { getCategories } from 'lib/bigcommerce'
 import { NextApiRequest, NextApiResponse } from 'next'
 
 export default async function handler(req: NextApiRequest, res: NextApiResponse) {
-  try {
-    const categories = await getCategories()
-    res.status(200).json(categories)
-  } catch (error) {
-    console.error('Failed to fetch categories', error)
-    res.status(400).json({ message: 'Failed to fetch categories' })
-  }
+  const categories = await getCategories()
```

Comment:
This is already logging errors, and if it throws this should be a 500, so I moved this to be not include the try catch